usr.bin/kdump/kdump.c

```diff
@@ -1609,6 +1609,23 @@ ktrsyscall_freebsd(struct ktr_syscall *ktr, register_t **resip,
 				narg--;
 				c = ',';
 				break;
+			case SYS_cheri_revoke:
+				putchar('(');
+				print_mask_arg(sysdecode_cr_flags, *ip);
+				ip++;
+				narg--;
+				c = ',';
+				/* XXX: don't need to print the rest? */
```

Comment:
Yes, you can drop these XXX.

What happens is that `ktrsyscall()` calls an ABI-specific routine (e.g. `ktrsyscall_freebsd()`) to decode any ABI-specific arguments.  The ABI routine returns the modified `ip` and `narg` to `ktrsyscall` which then loops one the remaining arguments printing them as plain numbers.  This loop in `ktrsyscall` handles system calls that weren't recognized by the ABI hook as well as any remaining arguments not consumed by the ABI hook.